arex-instrumentation/httpclient/arex-httpclient-apache-v4/src/main/java/io/arex/inst/httpclient/apache/common/ApacheHttpClientAdapter.java

```diff
@@ -99,28 +94,21 @@ public HttpResponseWrapper wrap(HttpResponse response) {
             return buildEmptyBodyResponseWrapper(response.getStatusLine().toString(), locale, headers);
         }
 
-        byte[] responseBody;
+        // Compatible with org.apache.http.impl.client.InternalHttpClient.doExecute
+        bufferResponseEntity(response);
+
+        byte[] responseBody = new byte[0];
```

Comment:
byte[] responseBody = new byte[0];

`ZERO_BYTE` instead of `new byte[0]` ?

Reply:
Fixed